modules/backend/lang/en/lang.php

```diff
@@ -141,7 +141,7 @@
             'name_field' => 'Name',
             'description_field' => 'Description',
             'is_new_user_default_field_label' => 'Default group',
-            'is_new_user_default_field_comment' => 'Add new administrators to this group by default',
+            'is_new_user_default_field_comment' => 'Add new administrators to this group by default.',
```

Comment:
I don't mind the minor formatting changes (only in language files), I'm just not going to go out of my way to make them. I'll merge them if PRs are made for them though; just not as sure about the changes to the periods.

As far as the on / off button width, I'd suggest making your translation changes and then creating an issue for the width of the buttons so that we can deal with it in the most efficient way possible after reviewing the issue.

Not sure what you're asking about the time format language variables